packages/@webex/plugin-meetings/src/meetings/util.ts

```diff
@@ -59,6 +59,22 @@ MeetingsUtil.handleRoapMercury = (envelope, meetingCollection) => {
         };
 
         meeting.mediaProperties.webrtcMediaConnection.roapMessageReceived(roapMessage);
+
+        let roapConnectionService;
+
+        try {
+          roapConnectionService = roapMessage.sdp
```

Comment:
can you write a comment on this? seems kinda whacky but i know it's to check which backend this is going to. 

I'm not sure there is really roapConnectionservice though... name might better be mediaServer?

Reply:
I would rather avoid having this code here and have that logic just in the test. You can check the SDP directly from the test by accessing `meeting.mediaProperties.webrtcMediaConnection.multistreamConnection.pc.getRemoteDescription().sdp` (it is a bit hacky because you will be accessing some private properties, but this is temporary solution anyway - we will not need it for long)

and I agree with Chris about the name, "mediaServer" is probably better

Reply:
Renaming this.

Reply:
I've added test constants to `./test/utils/constants`.